indico/util/spreadsheets.py

```diff
@@ -86,17 +88,19 @@ def generate_csv(headers, rows):
     return buf
 
 
-def _prepare_excel_data(data):
+def _prepare_excel_data(data, tz=None):
     if isinstance(data, (list, tuple)):
         data = ', '.join(data)
     elif isinstance(data, set):
         data = ', '.join(sorted(data, key=unicode.lower))
     elif is_lazy_string(data) or isinstance(data, Markup):
         data = unicode(data)
+    elif isinstance(data, datetime.datetime) and tz:
```

Comment:
I'd use `from datetime import datetime` and then just use `datetime` in the check here.